EVSL_1.0/SRC/landos.c

```diff
@@ -21,8 +21,8 @@
  *    @param[in] msteps number of Lanczos steps
  *    @param[in] npts number of sample points used for the DOS curve
  *    @param[in] *intv Stores the two intervals of interest \\
- *      intv[0:1] = [lambda_min, lambda_max]\\
- *      intv[2:3] = [a b] = interval where DOS is to be computed
+ *      intv[0:1] = [a b] = interval where DOS is to be computed
+ *      intv[2:3] = [lambda_min, lambda_max]\\
  *
```

Comment:
For example, in /TESTS/PLanN/LapPLanN.c, we have

` xintv[0] = a;
  xintv[1] = b;
  xintv[2] = lmin;
  xintv[3] = lmax;
  . . .
  LanDos(nvec, Mdeg, npts, xdos, ydos, &ecount, xintv);
`
The order of intv was actually wrong. So to have an easier life, let's just fix the meaning of intv[4] :)